app/components/MadeBySwapr.tsx

```diff
@@ -0,0 +1,86 @@
+'use client';
+
+import { useState, useEffect } from 'react';
+
+const cities = ['Stockholm', 'Porto', 'Barcelona'];
+
+const getRandomCityIndex = () => Math.floor(Math.random() * cities.length);
```

Comment:
Why random? Probably a loop would be better since random can show the same city for a long time or just show 2 of them.

Reply:
true, that's an issue, the idea was be more dynamic, but maybe it's better sequentially.